e2e-polybft/framework/test-cluster.go

```diff
@@ -333,18 +333,22 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 	}
 
 	for i := 1; i <= int(cluster.Config.ValidatorSetSize); i++ {
-		cluster.InitTestServer(t, i, true)
+		if cluster.Config.HasBridge && i == 1 {
```

Comment:
You can remove this if else and just have this line:

`cluster.InitTestServer(t, i, true, cluster.Config.HasBridge && i == 1)`